tests/unit/apps/test_auxiliary.py

```diff
@@ -130,9 +130,9 @@ def test_auxiliary_upgrade_plan_ussuri_to_victoria_change_channel(status, config
             ),
         ),
         UpgradeStep(
-            description=f"Wait 300 s for model {model.name} to reach the idle state.",
+            description=f"Wait 1800 s for model {model.name} to reach the idle state.",
```

Comment:
Non-blocking: I find it a bit weird to look at when there's a space between `<num>` and `s`. I would suggest removing it in both code and tests:
```suggestion
            description=f"Wait 1800s for model {model.name} to reach the idle state.",
```